app/component/map/generic-marker.cjsx

```diff
@@ -60,9 +60,14 @@ class GenericMarker extends React.Component
   shouldComponentUpdate: (nextProps) ->
     return nextProps.id != @props.id
 
+  openPopup: =>
+    @refs['marker'].getLeafletElement().openPopup()
+
   getMarker: ->
     <Marker map={@props.map}
+            ref="marker"
             position={lat: @props.position.lat, lng: @props.position.lon}
+            interactive={false}
```

Comment:
I would maybe add that as a comment. (or at least part of that). What do you think.

I have no other comments for pull request..